src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static TaskList taskList;
+    private static final boolean EXIT = false;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        boolean notExit;
```

Comment:
I think this variable is a bit ambiguous. Maybe replace with remain or stay, to avoid confusion with the EXIT constant